app/league/[leagueId]/entry/[entryId]/week/Week.test.tsx

```diff
@@ -0,0 +1,137 @@
+import React from 'react';
+import Alert from '@/components/AlertNotification/AlertNotification';
+import { AlertVariants } from '@/components/AlertNotification/Alerts.enum';
+import { toast } from 'react-hot-toast';
+import { onWeeklyPickChange } from './WeekHelper';
+import { createWeeklyPicks } from '../../../../../../api/apiFunctions';
```

Comment:
This should work. Test it out for all paths.
```suggestion
import { createWeeklyPicks } from '@/api/apiFunctions';
```

Reply:
sweet it works now for that path